apps/condo/domains/ticket/components/CallRecordCard.tsx

```diff
@@ -0,0 +1,73 @@
+import { CallRecord } from '@app/condo/schema'
+import { Row, Col } from 'antd'
+import { Gutter } from 'antd/es/grid/row'
+import dynamic from 'next/dynamic'
+import React, { CSSProperties, useMemo } from 'react'
+
+import { PhoneIncoming, PhoneOutgoing } from '@open-condo/icons'
+import { useIntl } from '@open-condo/next/intl'
+import { Card, Typography } from '@open-condo/ui'
+
+import { getDateRender } from '@condo/domains/common/components/Table/Renders'
+import { formatPhone } from '@condo/domains/common/utils/helpers'
+
+const DynamicAudioPlayer = dynamic(
+    () => import('@condo/domains/common/components/AudioPlayer').then((module) => module.AudioPlayer),
```

Comment:
Is this hack, because player operates with DOM directly?

Reply:
yes

Reply:
Is it possible to move this responsibility to player itself? Since we'll need to do this trick in any parent component :(

Reply:
Fixed, used async import in `useEffect` in `AudioPlayer`